docs/docs/main/references/FAQ.md

```
@@ -1,45 +1,45 @@
 # Frequently Asked Questions
 
-### Is BioNeMo Framework free to use?
+### Is BioNeMo Recipes free to use?
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)